### PR TITLE
[RFC] Added support for multiple checkbox selection

### PR DESCRIPTION
**Pull request by abhisheksavant**
@jskulski @alexindigo 
Still need to add checkbox and multiple checkbox docs

### CODE REVIEW

File: lib/functional_mixin.js [hunk 1]
```diff
@@ -187,6 +187,41 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.multipleCheckbox = function busterFunctionalMixin_multipleCheckbox(target, callback)
+  {
+    target = typeof target == 'string' ? this.$(target) : target;
+    // update target
+    function selectCheckbox(target)
```
Comment (jskulski) [inline lib/functional_mixin.js]: it's a little strange, imo, that you pass in an array and it only uses the first element. Maybe just pass in the first element?

File: test/test-multipleCheckbox.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        assert.calledWith(this._trigger, innerTarget, 'click');
+        assert.calledWith(this._trigger, innerTarget, 'mouseout');
+        assert.calledWith(this._trigger, innerTarget, 'blur');
+        assert.calledWith(this._trigger, innerTarget, 'focusout');
+        assert.calledWith(this._trigger, innerTarget, 'DOMFocusOut');
+
+        done();
+      });
+    }
+  });
```
Comment (purusho) [inline test/test-multipleCheckbox.js:165]: new line ?

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@
 npm-debug.log
 coverage
 node_modules
+*.idea
```
Comment (purusho) [inline .gitignore:7]: what is this ?

Comment (abhisheksavant) [inline .gitignore:7]: Its the phpstorm keeps adding these files, so wanted to aadd it to git ignore

**Pull request merged**